src/main/java/com/chat/yourway/exception/ApiErrorResponse.java

```diff
@@ -0,0 +1,14 @@
+package com.chat.yourway.exception;
+
+import lombok.Getter;
+import lombok.RequiredArgsConstructor;
+import org.springframework.http.HttpStatus;
+
+@Getter
+@RequiredArgsConstructor
+public class ApiErrorResponse {
```

Comment:
You guys created the same dtos, when first of you will merge his PR into main the second one should merge it into his branch and remove duplicated code